docs/getting-started/generating-a-mnemonic.en.md

```diff
@@ -51,6 +51,8 @@ Krux keeps track of every roll you enter and displays the cumulative string of o
 
 When you have entered your final roll, Krux will hash this string using SHA256 and output the resulting hash to the screen so that you can verify it for yourself.
 
+In the case a camera snapshot is used as source, the image bytes, which contain each pixel color information, will be hashed just like it is done with the dice rolls string.
```

Comment:
Would "..., which contain each pixel in rgb565 format, ..." (or similar) be more informative and still read well?